mars/deploy/oscar/local.py

```diff
@@ -152,7 +152,9 @@ def __init__(
         self._mem_bytes = mem_total() if mem_bytes == "auto" else mem_bytes
         self._cuda_devices = self._get_cuda_devices(cuda_devices, n_worker)
         self._subprocess_start_method = subprocess_start_method
-        self._config = load_config(config, default_config_file=DEFAULT_CONFIG_FILE)
+        self._config = load_config(
+            config, default_config_file=DEFAULT_CONFIG_FILE, backend=backend
+        )
```

Comment:
We don't needs this because the execution config merging will update the backend.

Reply:
It didn't update the config in `config["task"]["execution_config"]["backend"]`